src/core/lombok/javac/handlers/HandleVal.java

```diff
@@ -58,10 +58,11 @@ private static boolean eq(String typeTreeToString, String key) {
 	}
 	
 	@SuppressWarnings("deprecation") @Override
-	public void visitLocal(JavacNode localNode, JCVariableDecl local) {
+	public void endVisitLocal(JavacNode localNode, JCVariableDecl local) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 37. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.